ideas/README.md

```diff
@@ -22,13 +36,13 @@ You will pick up new knowledge as you finish workshops and seminars.
 
 Focus on applying that new knowledge in practice.
 
-### Examples
+#### Examples
 
 * Focus on getting names absolutely perfect, break out a thesaurus and choose the absolute most precise words for the problem at hand.
 * Is your code open for extension and closed for modification? Think of ways that it could be extended, and create plugin points
 * Do your classes and methods have one reason to change? Can you increase cohesion by refactoring?
 
-## Read
+### Read
 
 Pick up a book and read, here are a few that are invaluable
```

Comment:
One question I'd have in mind: is this listed in a specific order I should follow?